### PR TITLE
[Sagercaster] Reintroducing timestamp channel

### PR DESCRIPTION
**Pull request by clinique**
Forecast will be updated only if it as really changed.
Some documentation enhancement after this community [post](https://community.openhab.org/t/question-about-the-configuration-for-the-sager-binding/97141/23)

### CODE REVIEW

File: bundles/org.openhab.binding.sagercaster/README.md [hunk 1]
```diff
@@ -24,9 +24,11 @@ The binding itself does not require any configuration.
 
 | Name               | Type     | Description                                                              |
 |--------------------|----------|--------------------------------------------------------------------------|
-| location           | Location | Latitude and longitude of the desired weather forecast.                  |
+| location (*)       | Location | Latitude and longitude of the desired weather forecast.                  |
 | observation-period | int      | Minimum delay (in hours) before producing forecasts. Defaulted to 6.     |
 
+(*) Only latitude is used by the algorithm.
+
 ## Channels
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/README.md:32]: Please add the timestamp channel in documentation

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -142,7 +143,7 @@
 	<channel-type id="wind-direction">
 		<item-type>String</item-type>
 		<label>@text/windDirectionLabel</label>
-		<category>Wind</category>
+		<category>wind</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:146]: Please keep camelcase as it is like that everywhere in all bindings.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -174,27 +176,31 @@
...
-	<channel-type id="timestamp" advanced="true">
+	<channel-type id="timestamp">
 		<item-type>DateTime</item-type>
-		<label>@text/timestampLabel</label>
-		<description>@text/timestampDescription</description>
-		<category>Observation time</category>
-		<state readOnly="true"></state>
+		<label>@text/timestampChannelLabel</label>
+		<description>@text/timestampChannelDescription</description>
+		<category>time</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:183]: Please use camelcase.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 4]
```diff
@@ -163,6 +164,7 @@
 		<item-type>String</item-type>
 		<label>@text/trendLabel</label>
 		<description>@text/trendDescription</description>
+		<category>line</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:167]: This category is not listed in documentation: https://www.openhab.org/docs/concepts/categories.html#channel-categories

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 5]
```diff
@@ -174,27 +176,31 @@
...
+		</tags>
+		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="cloudiness">
 		<item-type>Number:Dimensionless</item-type>
 		<label>@text/cloudinessLabel</label>
 		<description>@text/cloudinessDescription</description>
-		<category>Clouds</category>
+		<category>sun_clouds</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:195]: Sun_Clouds

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 6]
```diff
@@ -174,27 +176,31 @@
...
 		<state min="0" max="100" pattern="%d %%"/>
 	</channel-type>
 
 	<channel-type id="rain-qtty" advanced="true">
 		<item-type>Number</item-type>
 		<label>@text/rainQttyLabel</label>
 		<description>@text/rainQttyDescription</description>
-		<category>Rain</category>
+		<category>rain</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:203]: Keep camelcase

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 7]
```diff
@@ -206,7 +212,7 @@
 		<item-type>Switch</item-type>
 		<label>@text/rainingLabel</label>
 		<description>@text/rainingDescription</description>
-		<category>Rain</category>
+		<category>rain</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:215]: Keep camelcase

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 8]
```diff
@@ -217,7 +223,7 @@
 		<item-type>Number</item-type>
 		<label>@text/beaufortLabel</label>
 		<description>@text/beaufortDescription</description>
-		<category>Wind</category>
+		<category>wind</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:226]: Keep camelcase

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml [hunk 9]
```diff
@@ -174,27 +176,31 @@
...
 
-	<channel-type id="timestamp" advanced="true">
+	<channel-type id="timestamp">
 		<item-type>DateTime</item-type>
-		<label>@text/timestampLabel</label>
-		<description>@text/timestampDescription</description>
-		<category>Observation time</category>
-		<state readOnly="true"></state>
+		<label>@text/timestampChannelLabel</label>
+		<description>@text/timestampChannelDescription</description>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:182]: timestampChannelLabel and timestampChannelDescription are not defined in the default properties file.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:167]: It exists and works : 
image

File: bundles/org.openhab.binding.sagercaster/README.md (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/README.md:32]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:146]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 3 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:183]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:167]: Camelcased.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 5 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:195]: Done

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 6 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:203]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 7 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:215]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 8 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:226]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 9 above)
Comment (clinique) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml:182]: Done.

File: bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml]: Yes, found here https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui.iconset.classic/src/main/resources/icons/line.png
Documentation is outdated !

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**